docs/superpowers/plans/2026-04-26-doppler-correction.md

```
@@ -0,0 +1,1493 @@
+    fn load_doppler_tracking_enabled_tolerates_non_bool() {
+        let config = make_config();
+        config.write(|v| {
+            v[KEY_DOPPLER_TRACKING_ENABLED] = serde_json::json!("not a bool");
+        });
+        // Falls back to the default (true), not a panic.
+        assert!(load_doppler_tracking_enabled(&config));
+    }
+```
+
+If `make_config()` doesn't exist as a helper in this file's tests, look at how other `load_*` tests in nearby files (e.g. `source_panel.rs`) construct their `Arc<ConfigManager>` — likely:
+```rust
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)